ulib/axlibc/Cargo.toml

```diff
@@ -23,49 +23,68 @@ crate-type = ["staticlib"]
 default = []
 
 # Multicore
-smp = ["axstd/smp"]
+smp = ["axfeat/smp", "arceos_posix_api/smp"]
 
 # Floating point/SIMD
-fp_simd = ["axstd/fp_simd"]
+fp_simd = ["axfeat/fp_simd", "arceos_posix_api/fp_simd"]
 
 # Memory
-alloc = ["axstd/alloc", "dep:axalloc"]
+alloc = ["axfeat/alloc", "arceos_posix_api/alloc", "axio/alloc"]
+alloc-tlsf = ["axfeat/alloc-tlsf"]
+alloc-slab = ["axfeat/alloc-slab"]
+alloc-buddy = ["axfeat/alloc-buddy"]
+paging = ["axfeat/paging"]
+tls = ["axfeat/tls"]
 
 # Multi-task
-multitask = ["axstd/multitask", "axtask/multitask"]
+multitask = ["axfeat/multitask", "arceos_posix_api/multitask"]
+sched_fifo = ["axfeat/sched_fifo"]
+sched_rr = ["axfeat/sched_rr"]
+sched_cfs = ["axfeat/sched_cfs"]
 
 # File system
-fs = ["axstd/fs", "fd"]
+fs = ["axfeat/fs", "fd", "arceos_posix_api/fs"]
+myfs = ["axfeat/myfs"]
 
 # Networking
-net = ["axstd/net", "dep:axnet", "fd"]
+net = ["axfeat/net", "fd", "arceos_posix_api/net"]
+dns = []
+
+# Display
+display = ["axfeat/display"]
+
+# Device drivers
+bus-mmio = ["axfeat/bus-mmio"]
+bus-pci = ["axfeat/bus-pci"]
+driver-ramdisk = ["axfeat/driver-ramdisk"]
+driver-ixgbe = ["axfeat/driver-ixgbe"]
+driver-bcm2835-sdhci = ["axfeat/driver-bcm2835-sdhci"]
+
+# Logging
+log-level-off = ["axfeat/log-level-off"]
```

Comment:
log features have been specified by `axfeat` and enabled in `scripts/make/features.mk`